dbt/scripts/utils/townships.py

```diff
@@ -0,0 +1,78 @@
+# Utilities for working with townships when exporting QC reports
+import dataclasses
+
+
+@dataclasses.dataclass
+class Township:
+    """Class that represents a township for the purposes of QC reporting"""
+
+    township_code: str
+    township_name: str
+    tri_code: str
+    tri_name: str
+
+    def is_reassessed_during(self, year: int) -> bool:
+        """Helper function to determine if a town in a given year is undergoing
+        triennial reassessment"""
+        # 2024 is the City reassessment year (tri code 1), so
+        # ((2024 - 2024) % 3) + 1 == 1, and so on for the other two tris
+        return str(((year - 2024) % 3) + 1) == self.tri_code
```

Comment:
This abstraction really seems like it should live in a `ccao` package, but absent that package, I think it's useful for the purposes of the `export` code.

Reply:
Yep! We can move it there once that package is finished.